src/store/index.ts

```diff
@@ -237,14 +237,15 @@ const {
     },
 
     setNestedVariables(state, nestedVariables: NestedVariables) {
+      const newNestedVars = nestedVariables;
       // Remove duplicates from the nested variables
-      nestedVariables.bar = [...new Set(nestedVariables.bar)];
-      nestedVariables.glyph = [...new Set(nestedVariables.glyph)];
+      newNestedVars.bar = [...new Set(nestedVariables.bar)];
+      newNestedVars.glyph = [...new Set(nestedVariables.glyph)];
 
       // Allow only 2 variables for the glyphs
-      nestedVariables.glyph.length = nestedVariables.glyph.length > 2 ? 2 : nestedVariables.glyph.length;
+      newNestedVars.glyph.length = newNestedVars.glyph.length > 2 ? 2 : newNestedVars.glyph.length;
```

Comment:
I think  you could clarify this line by using `Math.max()` instead of the ternary operator.

Reply:
Fixed in 84fc61c